### ISSUE TITLE
希望增加对mongo等数据库的支持

### ISSUE THREAD

**Issue by xdandyx** [Tencent/APIJSON#213]
**具体说下**
目前好像只支持几种关系型数据库，希望增加对mongo等非关系型数据库的支持

**为什么**
由于历史原因，项目内部使用mongodb，我们还有在django项目中使用djongo实现ORM，如果不过分追求事务一致性，mongo可看作mysql的超集，所以希望支持mongo

**Comment by TommyLemon** [comment]
对 AbstractSQLConfig 改造很大，或者需要写一个新的 SQLConfig 来适配非 SQL 的 CRUD 语法。目前专注于关系型数据库，暂时不会支持 SQL 以外的方式。可以提个建议
https://github.com/Tencent/APIJSON/issues/37

**Comment by TommyLemon** [comment]
@xdewx   apijson-mongodb，NoSQL 数据库 MongoDB 的 APIJSON 插件
https://github.com/APIJSON/apijson-mongodb
<img width="1531" alt="image" src="https://github.com/Tencent/APIJSON/assets/5738175/36406e20-b157-4121-bca7-280491462481">
